src/realm/util/logger.hpp

```diff
@@ -188,18 +190,19 @@ class AppendToFileLogger : public StreamLogger {
 /// A thread-safe logger. This logger ignores the level threshold of the base
 /// logger. Instead, it introduces new a LevelThreshold object with a fixed
```

Comment:
The threshold is not fixed anymore. And I assume thread safety is actually achieved through a mutex.